docs/docs/deployment/configuration/importing-and-exporting.md

```diff
@@ -5,7 +5,7 @@ sidebar_position: 110
 
 :::tip
 
-You need to be running Flagsmith version `2.28.2` or greater to be able to export and import your Flagsmith data.
+You need to be running Flagsmith version `2.28.4` or greater to be able to export and import your Flagsmith data.
```

Comment:
Should we add a note to upgrade Flagsmith to the latest version when migrating self-hosted->saas?

Reply:
Well, I think it might just work with 2.28.4, but I'm not sure. I would like to test this theory. If it doesn't work, we should add this, yes.